### ISSUE TITLE
More time taken compared to the orginal c inteference 

### ISSUE THREAD

**Issue by kinginsky** [thomaspark-pkj/pyyolo#24]
For a image with a size of 640*480, in my GPU 1070，the c interface ouputs 0.02s，but when i use the pyyolo interference, i get a value of 0.08s for one image. So the python  interface takes three times bigger than the original c interface function.  What's the reason?

**Comment by abhigoku10** [comment]
@kinginsky  yes even i am facing the same issue . do u have any alternate solution for it do let me know

**Comment by TheMikeyR** [comment]
I have tested the python interface, and made my own time print around pyyolo.detect() when comparing the time with the built-in prediction timer it says:
```
(builtin timer) Cam frame predicted in 0.017226 seconds.   
(python call timer) Time used predicting: 0.04249 seconds.
```

So about 3 times as long to do the entire process of calling detection, making the detection and then get the info inside python again. I wonder if there any way to speed this up for it get closer to the builtin timer
